docs/guides/admin/docs/workflowoperationhandlers/publish-engage-woh.md

```diff
@@ -30,6 +30,7 @@ Parameter Table
 |streaming-target-subflavors|Subflavor to use for distributed material                                                    |
 |merge-force-flavors        |Flavors of elements for which an update is enforced when merging catalogs.                  |
 |                           |Defaults to `dublincore/*,security/*`.
+|list-elements              |Include all media package elements in the publication. Defaults to `true`.
```

Comment:
Does it make sense to deactivate this? There is a | missing at the end.

Reply:
The consensus during the dev meeting seems to be to always enable listing the published elements.